rusk-abi/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.1.0] - 2021-02-19
+
+### Added
+
+- Add ABI infrastracture
+- Add Poseidon Hash host function
+- Add test contract
+- Add CHANGELOG.md
+- Add LICENSE
+- Add README.md
+
+[unreleased]: https://github.com/dusk-network/dusk-abi/compare/v0.1.0...HEAD
+[0.1.0]: https://github.com/dusk-network/dusk-abi/releases/tag/v0.1.0
```

Comment:
Not sure what;s this but is not inline with  the rest of our CHANGELOG formats.
Also, add newline please.

Reply:
It's our CHANGELOG format that is not inline with this. The other CHANGELOG claimed to use the "Keep a Changelog" but it's not. See: https://keepachangelog.com/en/1.0.0/

For example the section "Unreleased" is totally useless in that way. All the CHANGELOG I made are following the proper format, and makes all the version linked properly to the actual changes, released or not.

For example many CHANGELOG didn't use the proper date format and I already updated that to ISO 8601. Soon or later we should fix the other CHANGELOG, but it's a work that requires time. So at least for the new ones we're doing in the right way from the start.

Reply:
BTW about the new line, the CHANGELOG has one, so I'm not sure what you're referring to.